src/types/react-table-config.ts

```diff
@@ -149,5 +149,7 @@ declare module 'react-table' {
     extends UseExpandedRowProps<D>,
       UseGroupByRowProps<D>,
       UseRowSelectRowProps<D>,
-      UseRowStateRowProps<D> {}
+      UseRowStateRowProps<D> {
+    initialSubRows: Row<D>[];
```

Comment:
so this prop can be used by the user for the initial table state, right?
But then you use same prop for the actual sub rows in there.. Where the name `initial` naming is kinda confusing..but yeah, dont have better suggestion

Reply:
Nope, it is `Row` object. It has nothing to do with initial state. Here is the explanation: https://github.com/iTwin/iTwinUI-react/pull/266/files#diff-da3bb33793f0bcf735d6e37ffcb60a8d246b78615fb85e7c3c0a840c01e8d28bR21